src/components/network/network-renderer.js

```diff
@@ -664,6 +667,6 @@ genotet.NetworkRenderer.prototype.drawPath_ = function(terminator, coords,
         id: 'terminator',
         d: line([coords[0], coords[coords.length - 1]])
       });
-    this.svgPath_.select('path').remove();
+    this.svgPath_.selectAll('path').remove();
```

Comment:
Why not just remove the n-side polygon each time, before drawing the new polygon with n+1 sides? It seems to me that any time there only needs to be a single path on the screen.